x-pack/plugins/reporting/server/export_types/common/get_custom_logo.test.ts

```diff
@@ -34,8 +34,9 @@ test(`gets logo from uiSettings`, async () => {
   mockReportingPlugin.getUiSettingsServiceFactory = jest.fn().mockResolvedValue({
     get: mockGet,
   });
-
-  const { logo } = await getCustomLogo(mockReportingPlugin, headers, 'spaceyMcSpaceIdFace', logger);
+  const fakeRequest = mockReportingPlugin.getFakeRequest(headers, 'spaceyMcSpaceIdFace', logger);
```

Comment:
We will need to take mock reporting core usages out of the unit tests in `x-pack/plugins/reporting/server/export_types`.

For the fake request:
```
import { httpServerMock } from '@kbn/core-http-server-mocks';
const mockRequest = httpServerMock.createKibanaRequest();
```

For the UI Settings client:
```
import { uiSettingsServiceMock } from '@kbn/core/public/mocks';
const mockUiSettings = uiSettingsServiceMock.createStartContract();
```